soda_machine.rb

```diff
@@ -7,12 +7,25 @@ def initialize(args = {})
   end
 
   def current_inventory_count
+    @sodas.length
   end
 
   def find_soda(soda_brand)
+    @sodas.each do |soda|
+      if soda.brand == soda_brand
+        return soda
+      end
+    end
+      nil
   end
```

Comment:
You could refactor this like:
` def find_soda(soda_brand)
   @sodas.each do |soda|
      soda if soda.brand == soda_brand
   end
end`

#each already returns nil